File: Loops/fizzBuzz.js
function fizzBuzz (num){
    let finalArr=[]
    for(let i=1; i<=num; i++){
        if(i % 3===0 && i % 5===0){
            finalArr.push('FizzBuzz')
            continue;
        }
        if(i % 3===0){
            finalArr.push('Fizz')
            continue;
        }
        if(i % 5===0){
            finalArr.push('Buzz')
            continue;
        }
        finalArr.push(i.toString())
    }
    return finalArr;
}

console.log(fizzBuzz(3))

File: Loops_Hristo_Chipev/FizzBuzz.cs
using System;
namespace CSharp_Tasks
{
    public class FizzBuzz
    {
        public static string[] FizzBuzzM(int n)
        {
            string[] arr = new string[n];
            string res = "";

            for (int i = 0; i < n; i++)
            {
                if ((i + 1) % 3 == 0) res += "Fizz";
                if ((i + 1) % 5 == 0) res += "Buzz";

                if (res != "") arr[i] = res;
                else arr[i] = (i + 1).ToString();
                res = "";
            }

            return arr;
        }

    }
}

